src/lib.rs

```diff
@@ -794,6 +848,120 @@ pub struct ModuleConfig {
     config: serde_yaml::Value,
 }
 
+#[derive(Deserialize, Serialize, Debug, Clone, Default, TypedBuilder)]
+pub struct AllAppservicesConfig {
+    /// AppServices running in the guest.
+    #[serde(default)]
+    #[builder(default)]
+    guest: Vec<AppServiceConfig>,
+
+    /// AppServices running on the host.
+    #[serde(default)]
+    #[builder(default)]
+    host: Vec<AppServiceConfig>,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone, TypedBuilder)]
+pub struct AppServiceConfig {
```

Comment:
matrix-appservice-bridge requires additional fields in the top level, like `id`, could we accommodate for these like how we do in the `homeserver` section?

Reply:
I'll try and add that, yeah.

Reply:
Should be implemented in the latest version.